src/app/(routes)/page.tsx

```diff
@@ -45,11 +58,41 @@ export default function Home() {
                 passport.gitcoin.co
               </a>
             </li>
-            <li className="mb-2">Validate your identity up to a score of 30</li>
-            <li className="mb-2">"Bring Passport onchain" to Optimism network</li>
+            <li className="mb-2">
+              Validate your identity up to a{" "}
+              <a
+                href="https://support.gitcoin.co/gitcoin-knowledge-base/misc/explorer-passport-guide"
+                className="text-blue-500 underline"
+              >
+                score
+              </a>{" "}
+              of 30
+            </li>
+            <li className="mb-2">
+              "
+              <a
+                href="https://support.passport.xyz/passport-knowledge-base/using-passport/onchain-passport"
+                className="text-blue-500 underline"
+              >
+                Bring Passport onchain
+              </a>
+              " to Optimism network
+            </li>
           </ul>
+          {isConnected && isClient && <ValidatePassportScore address={address} chain={chain} onScoreChange={setScore} />}
```

Comment:
Let's add some call to action here for the case when user doesn't have the wallet connected. Something like "Please connect wallet to continue". Or maybe even move the "Connect Wallet" button here completely?